src/app/profile/payment-methods/payment-method/payment-method.tpl.html

```diff
@@ -0,0 +1,58 @@
+  <div class="panel panel-default panel-with-gray-background">
+    <div class="panel-heading" role="tab">
+      <div ng-class="{'collapsed': $ctrl.isCollapsed}" class="payment-summary-link p- collapse-indicator-row" role="button">
+        <div class="row">
+          <div class="col-md-8">
+            <p>
+              <!-- <img class="hidden-xs payment-icon" src={{"/assets/img/cc-icons/"+''}}> at the moment there is no way for me to know what image I need-->
+              <span class="payment-nickname">Visa</span>
```

Comment:
@adammeyer , I'm not sure what you mean. At the moment there is nothing that comes from API since the end-points are not returning what I add. It is dummy data for now.